cli.js

```diff
@@ -27,19 +36,34 @@ const cli = meow(`
 
 	Options
 	  --copy -c  Copy the first emoji to the clipboard
+	  --skin-tone -s  Set the default skin tone of the emoji (0 to 5)
 
 	Run it without arguments to enter the live search
+	Use Up/Down keys during live search to change the skin tones
 `, {
 	boolean: [
 		'copy'
 	],
 	alias: {
-		c: 'copy'
+		c: 'copy',
+		s: 'skinTone'
 	}
 });
 
+if (typeof cli.flags.skinTone === 'number') {
```

Comment:
> either all flags should have validation or none
Agreed!

@sindresorhus I think I didn't understand, setting the default value of skinTone to 0 for when the flag isn't given?

I've pushed a few more tweaks :simple_smile: